client/src/components/squids/SquidList.jsx

```diff
@@ -22,20 +24,25 @@ export const SquidList = () => {
   } else if (squidListQuery.isError) {
     squidListQueryOutput = squidListQuery.error.message
   } else {
-    squidListQueryOutput = squids.results.map((squid, index) => {
-      const hrElement = index > 0 ? <hr /> : ""
-      return (
-        <div key={squid.id}>
-          {hrElement}
-          <SquidTile squid={squid} />
-        </div>
-      )
-    })
+    squidListQueryOutput = squids.results.map((squid) => (
+      <div key={squid.id}>
+        <SquidTile squid={squid} />
+      </div>
+    ))
+  }
+
+  const queryClient = useQueryClient()
+  const useSquidRefetch = () => {
```

Comment:
the original `useQuery` should return a `refetch` method that allows you to initiate this request without having to know the query key